### ISSUE TITLE
Discriminator Loss converges to 0 while Generator loss pretty high

### ISSUE THREAD

**Issue by demiahmed** [lucidrains/lightweight-gan#133]
I am trying to train with a custom image dataset for about 600,000 epochs. At about halfway, my ```D_loss``` converges to ```0``` while my ```G_loss``` stays put at ```2.5```

My evaluation outputs are slowly starting to fade out to either black or white.

Is there any thing that I could to tweak my model? Either by increasing the threshold for the ```Discriminator``` or by training the ```Generator``` only?

**Comment by demiahmed** [comment]
Thanks for all the suggestions. I am trying out a combination of all measures. 

My default ```--gradient-accumulate-every``` is ```4```. Does higher gradient accumulation imitate a larger batch size?

**Comment by iScriptLex** [comment]
> Does higher gradient accumulation imitate a larger batch size?

Yes, it does. You can set `--gradient-accumulate-every 8` or even more.
